article_netmgr.cc

```diff
@@ -117,6 +117,10 @@ QNetworkReply * ArticleNetworkAccessManager::createRequest( Operation op,
   {
     QNetworkRequest newReq( req );
     newReq.setRawHeader("User-Agent", req.rawHeader("User-Agent").replace(qApp->applicationName(), ""));
+    if(useCustomUAHttpHeader)
```

Comment:
1. Move the existing `setRawHeader` statement above into an `else` block after this `if` block.
2. No need for braces for the `if` and the `else` if they contain only one statement.